### PR TITLE
Update light.mqtt.markdown

### PR DESCRIPTION
**Pull request by dchesterton**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#19675

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/light.mqtt.markdown [hunk 1]
```diff
@@ -82,6 +82,10 @@ brightness_value_template:
   description: "Defines a [template](/docs/configuration/templating/#processing-incoming-data) to extract the brightness value."
   required: false
   type: string
+color_temp_command_template:
+  description: "Defines a [template](/docs/configuration/templating/#processing-incoming-data) to compose message which will be sent to `color_temp_command_topic`. Available variables: `value`."
```
Comment (emontnemery) [inline source/_components/light.mqtt.markdown]: It's confusing to link to /docs/configuration/templating/#processing-incoming-data here, better just link to /docs/configuration/templating/
This should be fixed also for `rgb_command_template`
On the other hand, from color_temp_value_template, link to /docs/configuration/templating/#processing-incoming-data

Comment (dchesterton) [reply source/_components/light.mqtt.markdown]: Thanks, I've updated the links

Comment (emontnemery) [reply source/_components/light.mqtt.markdown]: I think you modified the link from `rgb_value_template`, instead of `rgb_command_template`?

**Comment by frenck** [comment]
@dchesterton, thank you for this PR!
The PR is against the wrong target base. It currently is targeted and based of the `current` branch. Since this PR has a code change pending, the documentation must go into the `next` branch, since it is depended on a future release of Home Assistant.

Please change the target branch to `next` and rebase the PR.

**Review by frenck — CHANGES_REQUESTED** [review]
Please rebase onto `next`.

File: source/_components/light.mqtt.markdown [hunk 2]
```diff
@@ -167,7 +171,7 @@ rgb_state_topic:
   required: false
   type: string
 rgb_value_template:
-  description: "Defines a [template](/docs/configuration/templating/#processing-incoming-data) to extract the RGB value."
+  description: "Defines a [template](/docs/configuration/templating/) to extract the RGB value."
```
Comment (emontnemery) [inline source/_components/light.mqtt.markdown]: This should still link to /docs/configuration/templating/#processing-incoming-data, change the link from `rgb_command_template instead`.

**Review by emontnemery — DISMISSED** [review]

**Comment by dchesterton** [comment]
Thanks for the feedback. I've fixed the links and rebased onto `next`.

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks, @dchesterton!
👍

**Pull request merged**